utility/src/main/java/org/oppia/android/util/accessibility/BUILD.bazel

```diff
@@ -0,0 +1,40 @@
+load("@io_bazel_rules_kotlin//kotlin:kotlin.bzl", "kt_android_library")
+load("@rules_jvm_external//:defs.bzl", "artifact")
+
+kt_android_library(
+    name = "accessibility_module",
+    srcs = ["AccessibilityModule.kt"],
+    visibility = ["//visibility:public"],
+    deps = [
+        ":android_accessibility_manager",
+        ":custom_accessibility_manager",
+        artifact("com.google.dagger:dagger:2.27"),
+        artifact("javax.inject:javax.inject:1"),
```

Comment:
Ditto here & below for the inject lines (see my comment in your other PR).

Reply:
Will do this after https://github.com/oppia/oppia-android/pull/2648#discussion_r573516626 is resolved